postgresql-simple.cabal

```diff
@@ -77,12 +77,12 @@ library
 
   -- GHC bundled libs
   build-depends:
-      base              >=4.6.0.0  && <4.15
+      base              >=4.7.0.0  && <4.15
     , bytestring        >=0.10.0.0 && <0.12
     , containers        >=0.5.0.0  && <0.7
     , template-haskell  >=2.8.0.0  && <2.17
     , text              >=1.2.3.0  && <1.3
-    , time              >=1.4.0.1  && <1.12
+    , time              >=1.9.0.0  && <1.12
```

Comment:
Just noticed that `time` only supports `CalendarDiffTime` from version 1.9 onwards. Since 1.9 was released in January 2018, I just bumped the version, but if using `#if MIN_VERSION_time` would be more appropriate, I'm open to doing that too.